cypress/integration/todo_edit.spec.js

```diff
@@ -0,0 +1,83 @@
+describe('Editing todos', function() {
+  beforeEach(function() {
+    /**
+     * `startSeeded` is a custom command that starts the app with seed data.
+     * Check out the source in ../support/commands.js
+     */
+    cy.startSeeded()
```

Comment:
is `cy` a "Global" that gets "injected" by the Cypress test runner?

Reply:
Yup! Cypress exposes the `cy` and `Cypress` objects as globals in your tests.